src/RestrictedTokenFaucet.sol

```diff
@@ -8,52 +8,60 @@ interface ERC20Like {
 contract RestrictedTokenFaucet {
     // --- Auth ---
     mapping (address => uint) public wards;
-    function rely(address guy) public isOwner { wards[guy] = 1; }
-    function deny(address guy) public isOwner { wards[guy] = 0; }
-    modifier auth { require(wards[msg.sender] == 1); _; }
-    modifier isOwner { require(owner == msg.sender); _; }
+    function rely(address guy) public auth { wards[guy] = 1; }
+    function deny(address guy) public auth { wards[guy] = 0; }
+    modifier auth {
+        require(wards[msg.sender] == 1, "token-faucet/no-auth");
+        _;
+    }
+    // --- Gulp Whitelist ---
+    mapping (address => uint) public list;
+    function hope(address guy) public auth { list[guy] = 1; }
+    function nope(address guy) public auth { list[guy] = 0; }
+
     address public owner;
```

Comment:
We don't need this anymore

Reply:
what do you mean here?

Reply:
we don't need the `address public owner` storage anymore with `auth` and `hope`/`nope`

Reply:
Ahh ok, will remove it now.